vnext/ReactWindowsCore/tracing/fbsystrace.h

```diff
@@ -51,44 +55,48 @@ class FbSystraceSection {
             .count());
   }
 
-  void init(std::string &&v) {
-    args_[index_++] = std::move(v);
-    begin_section();
+  template <typename... RestArg>
+  FbSystraceSection(uint64_t tag, std::string &&profileName, RestArg &&... rest)
+      : tag_(tag), profile_name_(std::move(profileName)) {
+    id_ = s_id_counter++;
+    init(std::forward<RestArg>(rest)...);
   }
 
-  void init(const std::string &v) {
-    args_[index_++] = v; // copy
-    begin_section();
+  ~FbSystraceSection() {
+    end_section();
   }
 
-  void init() {
-    begin_section();
+ private:
+  template <typename T>
+  using ConsructsToString =
+      std::bool_constant<std::is_convertible<T, std::string>::value || std::is_constructible<std::string, T>::value>;
+
+  template <typename Arg, typename std::enable_if_t<ConsructsToString<Arg>::value> * = nullptr>
```

Comment:
>> * = nullptr [](start = 81, length = 13)

I think the latest 'fashion' after STL's presentation is to write:
, int> = 0

Though, I prefer: '= 1' as it conveys 'true' semantic better than 0 or nullptr.